chart/epinio/templates/server.yaml

```diff
@@ -292,20 +292,20 @@ spec:
           livenessProbe:
             httpGet:
               path: /ready
-              port: 80
+              port: 8030
           name: epinio-server
           ports:
-            - containerPort: 80
+            - containerPort: 8030
           volumeMounts:
           - name: tmp-volume
             mountPath: /tmp
           readinessProbe:
             httpGet:
               path: /ready
-              port: 80
-      # securityContext:
-      #   runAsNonRoot: true
-      #   runAsUser: 65532
+              port: 8030
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1000
```

Comment:
Does this UID need to match an existing user in the base image? Also I'm reading here: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod

```
 If this field is omitted, the primary group ID of the containers will be root(0).
```
Maybe we need to set `runAsGroup` too?